megamek/src/megamek/common/Game.java

```diff
@@ -3273,8 +3278,77 @@ public Map<String, BehaviorSettings> getBotSettings() {
     public void setBotSettings(Map<String, BehaviorSettings> botSettings) {
         this.botSettings = botSettings;
     }
-
-    /**
+    
+    /**
+     * Place a carryable object on the ground at the given coordinates
+     */
+    public void placeGroundObject(Coords coords, ICarryable carryable) {
+    	if (!groundObjects.containsKey(coords)) {
```

Comment:
The entire method code can be written as
`groundObjects.computeIfAbsent(coords, k -> new ArrayList<>()).add(carryable);`
(not tested)